edb/server/server.py

```diff
@@ -1273,9 +1273,18 @@ async def _maybe_apply_patches(
                     patch_info = await bootstrap.gather_patch_info(
                         num, kind, patch, conn
                     )
+
+                    # Reload the compiler state from this database in
+                    # particular, so we can compiler from exactly the
+                    # right state. (Since self._std_schema and the like might
+                    # be further advanced.)
```

Comment:
```suggestion
                    # particular, so we can compile from exactly the
                    # right state. (Since self._std_schema and the like might
                    # be further advanced.)
```